tfhe/src/core_crypto/entities/seeded_lwe_ciphertext.rs

```diff
@@ -29,6 +29,25 @@ impl<T: UnsignedInteger> ParameterSetConformant for SeededLweCiphertext<T> {
     }
 }
 
+#[cfg(test)]
+impl<Scalar: UnsignedInteger> SeededLweCiphertext<Scalar> {
```

Comment:
same thing comment to explain why we have this

Reply:
Done